test/system/test_all.sh

```diff
@@ -30,13 +30,10 @@
 # THE POSSIBILITY OF SUCH DAMAGE.
 #;**********************************************************************;
 
-####before this script, please make sure all test scripts copy here, TPM is initialized.
-
 #!/bin/bash
-new_path=`dirname $0`
```

Comment:
We never needed this, we can just execute the script with a relative path so no need to set the script dir into PATH